README.md

```diff
@@ -27,9 +45,71 @@ While setting up cassandra instance, make sure the data directories of cassandra
 Testing
 -------
 
-We rely on docker containers for integration tests.
+The test framework is set up to run 4.1 and 5.0 (Trunk) tests (see `TestVersionSupplier.java`) by default.  
+You can change this via the Java property `cassandra.sidecar.versions_to_test` by supplying a comma-delimited string.
+For example, `-Dcassandra.sidecar.versions_to_test=4.0,4.1,5.0`.
 
-The only requirement is to install and run [Docker](https://www.docker.com/products/docker-desktop/) on your test machine.
+In order for tests to run successfully under JDK11, you'll need to add the following JVM arguments to your test runner of choice.
+You should also set your test framework to fork a new process at least every class, if not every method, as there are still
+a few unresolved memory-related issues in the in-jvm dtest framework.
+```
+-Djdk.attach.allowAttachSelf=true
+-XX:+UseConcMarkSweepGC
```

Comment:
It does not seem that I need to add those options individually to run integrationTest. They are configured as part of the `integrationTest` task in the build file already.

Reply:
Yeah - I originally wrote it before I fixed the integrationTest task - but if someone wants to run them outside of `Gradle` they may need them, so I wanted to have them documented.

Reply:
I'll also update the README to note it's not needed if you run via Gradle.

Reply:
Is it possible to drop those entirely? What is the scenario that we want to developers to run with other build tools?

Reply:
yeah, I think this is unnecessary. Don't we already have these in build.gradle? Someone interested in building it manually without using build tools can look inside build.gradle?